.chloggen/revertbatch.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: batchprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Revert "Add batchprocessor support for client metadata (#7578)"
```

Comment:
If this hasn't been released yet, shouldn't the revert just remove the changelog entry for the thing being reverted?

Reply:
0.78.0 is release

Reply:
time flies -- I thought this was added after the last release